doc/src/manual/multi-threading.md

```diff
@@ -372,6 +372,18 @@ threads in Julia:
     This may require some transitional work across the ecosystem before threading
     can be widely adopted with confidence. See the next section for further details.
 
+## [Task Migration](@id man-task-migration)
```

Comment:
Should this section mention the `:static` scheduler (not mentioned anywhere) as a way to recover the old behaviour?

Reply:
Makes sense. Let's do that in https://github.com/JuliaLang/julia/pull/48542 given this is a backport